presentation/scripts/lab1-1.sh

```diff
@@ -2,17 +2,17 @@
 
 source ./common.sh
 
-comment "Build the first version of our docker image"
+comment "Pull the first version of our docker image"
 DIR=$(pwd)
 cd ../../workshop/Lab1
-doit docker build --tag ${IMAGE_NAME}:v1 .
+doit docker pull ${IMAGE_NAME}:v1 
```

Comment:
do we want to explicitly pull from dockerhub `ibmcom/guestbook` instaed of IMAGE_NAME ?
then re tag it after?

Reply:
I think it depends on whether we expect people to have run the build step or not. If they do then it would be nice if they pulled from IBM's registry. Perhaps we should modify "common.sh" to have both IMAGE_NAME=.. lines and comment-out the IBM Cloud one. Then its easy for someone to switch based on what they've done before. We just need to update the README and/or common.sh file to tell them what to do.

Reply:
I don't know if we want to completely skip references to images.

But it is kind of nice in a way to remove dependency on container service/container registry.

Reply:
using IMAGE_NAME=ibmcom/guestbook